hokusai_onefile.spec

```diff
@@ -19,7 +19,7 @@ hidden_imports = ['configparser']
 a = Analysis(['bin/hokusai'],
              pathex=['.'],
              binaries=[],
-             datas=[('./hokusai/templates/*.j2', 'hokusai/templates/'), ('./hokusai/templates/.dockerignore.j2', 'hokusai/templates/'), ('./hokusai/templates/hokusai/*.j2', 'hokusai/templates/hokusai/'), ('./hokusai/VERSION', 'hokusai/')] + cert_datas,
+             datas=[('./hokusai/templates/*.j2', 'hokusai_datas/templates/'), ('./hokusai/templates/.dockerignore.j2', 'hokusai_datas/templates/'), ('./hokusai/templates/hokusai/*.j2', 'hokusai_datas/templates/hokusai/'), ('./hokusai/VERSION', 'hokusai_datas/')] + cert_datas,
```

Comment:
this is because `brew`'s `prefix.install` insists that the name of the executable (`hokusai`) to be symlinked must literally exist in the bundle, thus conflicting with the `datas` folder name.